style.css

```diff
@@ -1,448 +1,1103 @@
+    width: 90%;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+  }
+
+  .h003 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+
+    /* or 140% */
+
+    text-align: center;
 
-.submit_button button:hover {
-  padding: 10px 15px;
-  background-color: #6070ff;
-  color: #dfe1e6;
+    /* Primary/B50 */
+
+    color: #ebebff;
+  }
+
+  .input {
+    background: #fff;
+
+    /* Base/100 */
+    font-size: 16px;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    height: 3rem;
+    margin-top: 1rem;
+    width: 98%;
+    padding-left: 1rem;
+  }
+
+  .mssg {
+    background: #fff;
+
+    /* Base/100 */
+    padding-left: 1rem;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    height: 8rem;
+    font-size: 16px;
+    margin-top: 1rem;
+    width: 98%;
+  }
+
+  .formy-btn {
+    padding: 1rem 2rem 1rem;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    box-sizing: border-box;
+    gap: 12px;
+    background: #fff;
+    border: 1px solid #435c9d;
+    border-radius: 14px;
+    margin-top: 3rem;
+    transition: color 0.5s ease;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 25%;
+  }
+
+  .formy-btn:hover {
+    background: skyblue;
+    color: white;
+  }
+
+  nav .ul002 li a {
+    display: none;
+  }
+
+  .cards1 {
+    display: none;
+  }
+
+  .contact h3 {
+    font-family: "Poppins", sans-serif;
+    padding-top: 30px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-top: -5%;
+    display: flex;
+    padding-left: 15px;
+    align-self: baseline;
+    text-decoration: none;
+    margin-bottom: 2%;
+  }
+
+  .my001 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
 }
 
-.submit_button button:active {
-  padding: 10px 15px;
-  background-color: #001aff;
-  color: #fff;
+@media only screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    box-sizing: border-box;
+  }
+
+  .sp2 img {
+    display: none;
+  }
+
+  .ul002 {
+    float: right;
+    display: flex;
+    margin-top: 0%;
+    margin-right: 10%;
+  }
+
+  li {
+    list-style-type: none;
+    margin-left: 2%;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .ul002 > li > a {
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: black;
+    padding-right: 25px;
+  }
```

Comment:
- [ OPTIONAL] I noticed the `color` of the links is `#344563` and not black. It would be nice to implement it so that your design looks very identical to the Figa template you chose. 👍